UnitTests/UnitTests.Shared/Mvvm/Test_Messenger.Request.cs

```diff
@@ -20,7 +20,7 @@ public void Test_Messenger_RequestMessage_Ok()
             var messenger = new Messenger();
             var recipient = new object();
 
-            void Receive(NumberRequestMessage m)
+            void Receive(object recipient, NumberRequestMessage m)
```

Comment:
We should check we're getting the right `recipient` value too, eh?

Reply:
Yup, right! Done in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3424/commits/93456ae0b72dbf51eda4ebe2b3380062af019ddd 😊